src/shared/requestListeners.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { localize } from 'vscode-nls'
+import { CredentialsStore } from '../credentials/credentialsStore'
+import { LoginManager } from '../credentials/loginManager'
+import { ServiceOptions } from './awsClientBuilder'
+import { CredentialsProfileMru } from './credentials/credentialsProfileMru'
+import globals from './extensionGlobals'
+
+export const CREDENTIAL_ERROR_REQUEST_LISTENER: ServiceOptions = {
```

Comment:
If we're calling this file `requestListeners`, then it'd make more sense to export things with the `RequestListener` shape, not `ServiceOptions`